src/Service/VendorService/PressReader/PressReaderVendorService.php

```diff
@@ -0,0 +1,135 @@
+<?php
+
+/**
+ * @file
+ * Get cover from PressReader based on data well searches.
+ */
+
+namespace App\Service\VendorService\PressReader;
+
+use App\Service\VendorService\DataWell\DataWellSearchService;
+use App\Service\VendorService\ProgressBarTrait;
+use App\Service\VendorService\VendorImageValidatorService;
+use App\Service\VendorService\VendorServiceInterface;
+use App\Service\VendorService\VendorServiceTrait;
+use App\Utils\Message\VendorImportResultMessage;
+use App\Utils\Types\IdentifierType;
+use App\Utils\Types\VendorStatus;
+
+/**
+ * Class PressReaderService.
+ */
+class PressReaderVendorService implements VendorServiceInterface
+{
+    use ProgressBarTrait;
+    use VendorServiceTrait;
+
+    protected const VENDOR_ID = 19;
+    private const VENDOR_ARCHIVE_NAME = 'pressreader';
+
+    private DataWellSearchService $datawell;
+
+    private string $urlPattern = 'https://i.prcdn.co/img?cid=%s&page=1&width=1200';
```

Comment:
All magic values should be class constants

Reply:
Fixed